hedera/settings.py

```diff
@@ -8,17 +8,20 @@
 from .aws import get_ecs_task_ips
 
 
+IS_LTI = bool(os.environ.get("IS_LTI"))
```

Comment:
Shell environment variable values are read in as strings. Accordingly, the expression `bool(os.environ.get("IS_LTI"))` will always evaluate to `True`, except when the env variable is an empty string, i.e. `IS_LTI=""`. Concretely, even if the env variable `IS_LTI=0` or `IS_LTI=False`, the expression `bool(os.environ.get("IS_LTI"))` will still evaluate to `True`. That would be undesired behavior.

To fix this, one might try:
```
IS_LTI = bool(int(os.environ.get("IS_LTI", "0")))
```
This way, the setting `IS_LTI` will be `True` if and only if the env variable `IS_LTI=1`. Note that a `ValueError` will be thrown if the env var value is not a string that can convert to type `int`. So, a better solution then becomes:
```
try:
    IS_LTI = bool(int(os.environ.get("IS_LTI", "0")))
except ValueError:
    IS_LTI = False
```
Or, succinctly:
`IS_LTI = (os.environ.get("IS_LTI") == "1")`

Reply:
Yeah, I was following the pattern already set in this file for reading env vars. The way that I typically do it is:

`IS_LTI = os.environ.get("IS_LTI", False)` knowing that if I want the value to be false, it has to be set as `IS_LTI=` or removed totally in the environment.